curvefs/devops/templates/client-conf/client.conf

```diff
@@ -0,0 +1,68 @@
+# mdsOpt
```

Comment:
If here are another conf, will have difficulty to maintain it and curvefs/conf. This is also a problem  in 'curve'. Can improve?

Reply:
> If here are another conf, will have difficulty to maintain it and curvefs/conf. This is also a problem in 'curve'. Can improve?

We can add specified filed for every config file in `curvefs`/conf, and let ansible to generate template file by automitic, e.g:
```
curvefs/conf/mds.conf

mds.dummy.port=7777  # __TMPL__ {{ curvefs_mds_listen_dummy_port }} __TMPL__

metaserver.addr=127.0.0.1:16701  # __TMPL__
{{ groups.metaserver | join_peer(hostvars, "metaserver_listen_port") }}
# __TMPL__
```